### DISCUSSION TITLE
is there a way to not click on some ads?

### DISCUSSION THREAD

**Discussion by Trussmeister** [dhowe/AdNauseam#2542] [Q&A] [answered]
I dont know if the filters tab supports this and i cant find anything about it on the web, thanks for reading.

**Comment by mneunomne** [comment, marked as answer]
@Trussmister we currently don't support a way to custumise what ads are clicked or not. But perhaps it is something we can think about implementing as a custom rule in "My Rules" tab. Thank you for your question.

**Reply by Trussmeister** [reply]
Thanks
